openedx/features/student_certificates/views.py

```diff
@@ -111,14 +113,28 @@ def student_certificates(request):
             course_title = course.display_name
 
         user_certificates.append({
-            'course_name': course_name,
-            'course_title': course_title,
+            'certificate_name': course_name,
+            'certificate_title': course_title,
             'social_sharing_urls': get_philu_certificate_social_context(course, certificate),
             'certificate_url': "%s%s" % (settings.LMS_ROOT_URL, certificate_url),
             'course_start': start_date.strftime('%b %d, %Y') if start_date else None,
             'completion_date': completion_date.strftime('%b %d, %Y') if completion_date else None,
+            'is_program_cert': False,
         })
 
+    program_credentials = get_credentials(request.user, credential_type='program')
+    for credential in program_credentials:
+        program_name = [attribute['value'] for attribute in credential.get('attributes',{})
+                        if 'name' in attribute and attribute['name'] == 'program_name']
+        if program_name:
+            user_certificates.append({
+                'certificate_name': program_name[0],
+                'certificate_title': program_name[0],
+                'certificate_url': credential.get('certificate_url'),
+                'completion_date': date_from_str(credential.get('created')).strftime(COMPLETION_DATE_FORMAT),
```

Comment:
Please consider following comment again
You can define a method to convert date to string by giving string format. just like we define date_from_str..

We can move date_to_str and date_from_str to philu_utils.py so that can be used in other apps as well